Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "yorick_jit_runtime"
+version = "0.1.0"
+authors = ["Laurence Tratt <laurie@tratt.net>"]
```

Comment:
Can we add me as well? I know this is all your code, but I guess we required my (failed) attempt to arrive here.

Reply:
In retrospect, we should probably do `Software Development Team <info@soft-dev.org>`?

Reply:
If you wish, but I'd prefer the individual members to be listed, as team membership is in flux, and I'm sure people would appreciate credit for past work after they have moved on.

Reply:
We don't do it in the copyright notices, so it seems a bit inconsistent to me. But I can be persuaded.

Reply:
Hrm, let's think about this. I think you can see my point though?

Reply:
Yes, but then we get to the question of "what constitutes an author". I think this field is a bit of a minefield :/